src/commands/game_commands/lookup.ts

```diff
@@ -39,11 +44,290 @@ const getDaisukiLink = (id: string, isMV: boolean): string => {
+        );
+    }
+
+    const viewsString = LocalizationManager.localizer.translate(
+        guildID,
+        "misc.views"
+    );
+
+    const fields = [
+        {
+            name: viewsString[0].toUpperCase() + viewsString.slice(1),
+            value: friendlyFormattedNumber(views),
+        },
+        {
+            name: LocalizationManager.localizer.translate(
+                guildID,
+                "misc.releaseDate"
+            ),
+            value: friendlyFormattedDate(publishDate, guildID),
+        },
+        {
+            name: LocalizationManager.localizer.translate(
+                guildID,
+                "misc.songAliases"
+            ),
+            value:
+                songAliases ||
+                LocalizationManager.localizer.translate(guildID, "misc.none"),
+        },
+        {
+            name: LocalizationManager.localizer.translate(
+                guildID,
+                "misc.artistAliases"
+            ),
+            value:
+                artistAliases ||
+                LocalizationManager.localizer.translate(guildID, "misc.none"),
+        },
+    ];
+
+    if (kmqSongEntry) {
+        fields.push(
+            {
+                name: LocalizationManager.localizer.translate(
+                    guildID,
+                    "misc.duration"
+                ),
+                value:
+                    songDuration ||
+                    LocalizationManager.localizer.translate(
+                        guildID,
+                        "misc.notApplicable"
+                    ),
+            },
+            {
+                name: LocalizationManager.localizer.translate(
+                    guildID,
+                    "command.lookup.inCurrentGameOptions"
+                ),
+                value: LocalizationManager.localizer.translate(
+                    guildID,
+                    includedInOptions ? "misc.yes" : "misc.no"
+                ),
+            }
+        );
+    }
+
+    sendInfoMessage(messageContext, {
+        title: `${songName} - ${artistName}`,
+        url: `https://youtu.be/${videoID}`,
+        description,
+        thumbnailUrl: `https://img.youtube.com/vi/${videoID}/hqdefault.jpg`,
+        fields: fields.map((x) => ({
+            name: x.name,
+            value: x.value,
+            inline: true,
+        })),
+    });
+}
+
+async function lookupBySongName(
+    songName: string,
+    locale: LocaleType,
+    message: GuildTextableMessage,
+    guildID: string
+): Promise<void> {
+    const messageContext = MessageContext.fromMessage(message);
+    const kmqSongEntries: QueriedSong[] = await dbContext
+        .kmq("available_songs")
+        .select(SongSelector.getQueriedSongFields())
+        .whereRaw(`song_name_en LIKE "%${songName}%"`)
+        .orWhereRaw(`song_name_ko LIKE "%${songName}%"`)
+        .orderByRaw("CHAR_LENGTH(song_name_en) ASC")
+        .orderBy("views", "DESC")
+        .limit(100);
```

Comment:
Will the number of results impact performance?

Reply:
At 100, no